packages/mui-codemod/src/v6.0.0/sx-prop/test-cases/sx-dynamic2.actual.js

```diff
@@ -0,0 +1,23 @@
+<Accordion
+  expanded={expanded}
+  onChange={handleExpansion}
+  slotProps={{ transition: { timeout: 400 } }}
+  sx={{
+    '& .MuiAccordion-region': { height: expanded ? 'auto' : 0 },
+    '& .MuiAccordionDetails-root': { display: expanded ? 'block' : 'none' },
+  }}
+></Accordion>;
+
+<Box
+  sx={{
```

Comment:
Can we have an example where sx is an array as an input?

Reply:
Good one.

Reply:
Added